src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services9_compiled.php

```diff
@@ -246,7 +246,7 @@ protected function getMethodCall1Service()
 
         $instance->setBar($this->get('foo'));
         $instance->setBar(NULL);
-        $instance->setBar(($this->get("foo")->foo() . (($this->hasparameter("foo")) ? ($this->getParameter("foo")) : ("default"))));
+        $instance->setBar(($this->get('foo')->foo().(($this->hasParameter('foo')) ? ($this->getParameter('foo')) : ('default'))));
```

Comment:
@stof : this is code style changes, triggering all those fixtures adjustments - PR simply marked as not fulfilling code style requirements.

The same in Symfony/Component/DependencyInjection/Tests/Fixtures/php/services9.php